dependencies.yaml

```diff
@@ -700,3 +704,18 @@ dependencies:
           - matrix: null
             packages:
               - libucx>=1.15.0
+  depends_on_nccl:
+    common:
+      - output_types: conda
+        packages:
+          - &nccl_unsuffixed nccl>=2.19
```

Comment:
Is 2.19 a high enough lower bound? The latest is 2.26. I'm pretty sure we rely on newer versions than 2.19 for Blackwell support and other features. I don't know the exact bound to use here but we should do some validation with the oldest NCCL release we claim to support.

Reply:
That's a good question. I think we could bump the version of NCCL as a follow-on to this PR depending on the . I just used the version constraint that was already present. I just checked our `ci-wheel` images and the system version on those is also `2.26`.

Reply:
Alrighty! If this matches other bounds already being used in RAFT, we can merge in the current state.